README.md

```diff
@@ -1,35 +1,59 @@
 ![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
 
-## CyberArk Ansible Conjur Collection
+# CyberArk Ansible Conjur Collection
 
-****
+This collection contains plugins to be used for CyberArk Conjur & DAP (Dynamic Access Provider) hosted in [ansible galaxy](https://galaxy.ansible.com/cyberark/conjur).
 
-### cyberark.conjur
+## Table of Contents
+- [CyberArk Ansible Conjur Collection](#cyberark-ansible-conjur-collection)
+  * [Requirements](#requirements)
+  * [conjur_variable Lookup Plugin](#conjur_variable-lookup-plugin)
+    + [Role Variables](#role-variables)
+    + [Dependencies](#dependencies)
+    + [Example Playbook](#example-playbook)
+  * [Conjur Ansible Role](#conjur-ansible-role)
```

Comment:
can the order of these two be swapped? usually I think you need to use the role before you can use the lookup plugin